pom.xml

```diff
@@ -79,13 +79,17 @@
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
             <version>1.1.3</version>
-            <scope>test</scope>
```

Comment:
not sure we need this in run-time, SonarQube probably has this dependency already